package.json

```diff
@@ -113,9 +112,9 @@
   "dependencies": {
     "ember-cli-babel": "^5.1.8",
     "ember-cli-htmlbars": "^1.0.3",
-    "ember-cli-sass": "^5.2.0"
+    "ember-cli-sass": "^5.6.0"
```

Comment:
Why make the minimum version higher, does it not work with `5.2.0` anymore? If that is not the case please keep the version lower.